rca/guides/migrations/0010_stop_allowing_blank_pages.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.12 on 2021-01-22 14:25
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("guides", "0009_allow_blank_values"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="guidepagerelatedpages",
+            name="page",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
```

Comment:
I'm in two minds weather to set the to PROTECT...

Reply:
I think CASCADE is fine because this is just a related page placement. If the page needs to be deleted, it'd be annoying to have to go into every page where its used to unset. I think it'd 500 error with PROTECT anyway so the user wouldn't even be informed which ones.